Core/VariantManager.swift

```diff
@@ -21,16 +21,14 @@ import Foundation
 
 public enum FeatureName {
     
-    case homeRowOnboarding, homeRowReminder
+    case homeRowCTADefault, homeRowCTAAlternative1, homeRowCTAAlternative2
     
 }
 
 public struct Variant {
     
-    public static let defaultVariants = [
-        Variant(name: "mb", percent: 50, features: []),
-        Variant(name: "mc", percent: 25, features: [.homeRowOnboarding]),
-        Variant(name: "md", percent: 25, features: [.homeRowOnboarding, .homeRowReminder]),
+    public static let defaultVariants: [Variant] = [
```

Comment:
Do you need to migrate existing users' variants to a default state? e.g., users who were previously `mc`, should they now lose that variant?

Reply:
If I understand you, that's been done in #332

Reply:
Yeah, that's what I was asking.